release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -536,6 +536,13 @@ For more information, see xref:../networking/dns-operator.adoc#nw-controlling-dn
 ==== Provider networks support for clusters that run on {rh-openstack}
 {product-title} clusters on {rh-openstack-first} now support provider networks for all deployment types.
 
+[id="ocp-4-8-configurable-headerBufferMaxRewriteByte-headerBufferBytes-parameters"]
+==== Configurable tune.maxrewrite and tune.bufsize for HAProxy
+
+Cluster Administrators can now set `headerBufferMaxRewriteByte` and `headerBufferBytes` Ingress Controller parameters to configure `tune.maxrewrite `and `tune.bufsize` HAProxy memory options per-Ingress Controller.
```

Comment:
```suggestion
Cluster Administrators can now set `headerBufferMaxRewriteByte` and `headerBufferBytes` Ingress Controller tuning parameters to configure `tune.maxrewrite `and `tune.bufsize` HAProxy memory options per-Ingress Controller.
```

Reply:
Might want to be explicit here since these new API fields fall under the tuning options field.

Reply:
Done! Thank you!